src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/AuthorizationFunctionalTest.java

```diff
@@ -108,8 +117,7 @@ public static String generateRandomEmail() {
 
     @AfterAll
     public static void destroy() {
-        emailsTobeDeleted.forEach(email -> idamOpenIdClient.deleteSidamUser(email));
-        log.info("delete idam user called");
+
     }
 
```

Comment:
Are we no longer cleaning up test data? And if so, do we need this empty method?

Reply:
we can remove this method.